src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/_docker_utils.py

```diff
@@ -35,7 +35,7 @@
 
 EMPTY_GUID = '00000000-0000-0000-0000-000000000000'
 ALLOWED_HTTP_METHOD = ['get', 'patch', 'put', 'delete']
-ACCESS_TOKEN_PERMISSION = ['*', 'pull']
+ACCESS_TOKEN_PERMISSION = ['pull', 'push', 'delete', 'push,pull', 'delete,pull']
```

Comment:
This should really just be `['pull', 'push', 'delete']` and access a space-separated list of permissions. The service could either validate which combinations were valid or client-side validation, but we don't allow comma-separated lists in the CLI.

Reply:
We probably can just remove this list and client-side validation. The client has to send `push,pull` as one word as the service didn't provide a way to specify a list.

Reply:
Since we do not allow comma separated lists, you would need to expose it as a list and then use a type callable or validator to convert to the appropriate values the service accepts.

CLI
`--permissions push pull`

Service
`"push,pull"`

Reply:
Alternatively, you could keep the values list, and just call "push,pull" as "push-pull".  Then you have a type callable that replaces "-" with "," to send the properly formatted value to the service.

Reply:
I see where the confusion was. This is an internal method and we don't expose permissions to customers in the CLI.

Reply:
Ah, okay. In that case it is fine.